src/assets/js/search.js

```diff
@@ -98,18 +98,11 @@
   }
 
   const getExcerpt = (content, searchQuery) => {
-    const fullRegex = new RegExp(searchQuery, 'ig')
-    const fullSearchTerm = {
-      term: searchQuery,
-      regex: fullRegex,
-      matchIndex: content.search(fullRegex),
-    }
-
-    const searchTerms = searchQuery.split(' ').filter(i => i).map(term => {
-      const regex = new RegExp(term, 'ig')
-
-      return { term, regex, matchIndex: content.search(regex) }
-    })
+    const fullSearchTerm = getTermMatcher(searchQuery, content)
+    const searchTerms = searchQuery
+      .split(" ")
+      .filter((i) => i)
+      .map((term) => getTermMatcher(term, content));
```

Comment:
Per [this recent comment](https://github.com/dxw/playbook/pull/1055#discussion_r1095849745), it would be good to keep new code in line with [Standard](https://standardjs.com) formatting guidelines, so here that would mean using single quotes and dropping the semi-colon

If you're using VSCode, there's an extension and some settings that I can share that help make this automated on save - I had to play around quite a bit to get it working properly!

Reply:
Thanks for sharing this, have reformatted using standard, and also squashed the commits